### DISCUSSION TITLE
Changing the theme font

### DISCUSSION THREAD

**Discussion by linux-lukas** [luizdepra/hugo-coder#693] [General]
Can $user who uses a Hugo Page with this theme also change the font of the theme without further ado. Is this covered by the MIT licence?

> *"Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:"* 

@luizdepra

**Comment by luizdepra** [comment]
Sure, you are allowed to make any kind of change you wish. Also, you could fork the project and release it with your changes. MIT License only enforces that you need to keep a copy of the original license and copyright notice within your project.

```
The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
```
